app/src/components/report/ReportEdit.tsx

```diff
@@ -0,0 +1,209 @@
+                <Form.Control
+                    as="textarea"
+                    rows={3}
+                    placeholder="Please enter your issue description"
+                    defaultValue={report?.issueDescription}
+                    onChange={(event) => updateReport({ issueDescription: event.target.value })}
+                />
+            </Form.Group>
+
+            <Form.Group className="mb-3" controlId="EditReport.Attachments">
+                <Form.Label>Attachments</Form.Label>
+                <Form.Control
+                    type="text"
+                    placeholder="Please enter your attachments"
+                    defaultValue={report?.attachments}
+                    onChange={(event) => updateReport({ attachments: event.target.value })}
+                />
+            </Form.Group>
+
+            <Form.Label>Status Updates</Form.Label>
+
+            <Form.Group className="mb-3" controlId="EditReport.Email">
+                <Form.Check
+                    type="checkbox"
+                    label="Email"
+                    checked={report?.email}
+                    onChange={(event) => {
+                        updateReport({
+                            email: event.target.checked,
+                        });
+                    }}
+                />
+            </Form.Group>
+
+            <Form.Group className="mb-3" controlId="EditReport.Sms">
+                <Form.Check
+                    type="checkbox"
+                    label="SMS"
+                    checked={report?.sms}
+                    onChange={(event) => {
+                        updateReport({
+                            sms: event.target.checked,
+                        });
+                    }}
+                />
+            </Form.Group>
+
+            <Form.Group className="mb-3" controlId="EditReport.StatusOfReport">
+                <Form.Label>Status Of Report</Form.Label>
+                {['Submitted', 'In Progress', 'Completed', 'On Hold', 'Rejected'].map(
+                    (status) => (
+                        <Form.Check
+                            type="radio"
+                            label={status}
+                            name="status"
+                            checked={report?.statusOfReport === status}
+                            onChange={() => updateReport({ statusOfReport: status })}
+                        />
+                    )
+                )}
+            </Form.Group>
+
+            <Form.Group className="mb-3" controlId="EditReport.DateTimeOfSubmission">
+                <Form.Label>Date Time Of Submission</Form.Label>
+                <Form.Control
+                    type="text"
+                    readOnly={true}
+                    defaultValue={report?.dateTimeOfSubmission}
+                    onChange={(event) => updateReport({ dateTimeOfSubmission: event.target.value })}
+                />
+            </Form.Group>
```

Comment:
Is there a reason to include this? To me, it seems like a confusing/unnecessary step for our users - Shouldn't the API automatically set the time of submission?